docs/pages/cheatsheet.adoc

```diff
@@ -653,9 +965,38 @@ Removed from service object.
 |guard          | boolean expression    | The condition to allow the external user to access the services.
 |====================
 
+*Example:*
+
+[source,jsl]
+----
+actor human CustomerActor(Customer customer) {
+	menu CartView myCart <= customer label:"My cart";
+	menu ProductListView products label:"Products";
+}
+----
+
+=== Actor access
+
+Access provides an entity or collection of entities to the user represented by the actor.
```

Comment:
> Access provides

Suggestions:
- Access exposes...
- Access provides access to...
- (Or simply) Provide access to...

Reply:
done